test/suite/test_tiered16.py

```diff
@@ -31,7 +31,7 @@
 
 from helper_tiered import TieredConfigMixin, gen_tiered_storage_sources
 from wtscenario import make_scenarios
-import os, wiredtiger, wttest
+import errno, os, wiredtiger, wttest
```

Comment:
I don't see the `errno` package used here.

Reply:
Fixed